data-source/blocks/individual/identity-and-health/arrive_in_uk.jsonnet

```diff
@@ -36,51 +78,5 @@ local proxyTitle = {
       when: [rules.proxyYes],
     },
   ],
-  routing_rules: [
-    {
-      goto: {
-        block: 'length-of-stay',
-        when: [
-          {
-            id: 'arrive-in-uk-answer',
-            condition: 'greater than',
-            date_comparison: {
-              value: std.extVar('census_date'),
-              offset_by: {
-                years: -1,
-              },
-            },
-          },
-        ],
-      },
-    },
-    {
-      goto: {
-        block: 'when-arrive-in-uk',
-        when: [
-          {
-            id: 'arrive-in-uk-answer',
-            condition: 'equals',
-            date_comparison: {
-              value: std.extVar('census_date'),
-              offset_by: {
-                years: -1,
-              },
-            },
-          },
-        ],
-      },
-    },
-    {
-      goto: {
-        block: 'understand-welsh',
-        when: [rules.regionWales],
-      },
-    },
-    {
-      goto: {
-        block: 'language',
-      },
-    },
-  ],
+  routing_rules: routingRules(region_code),
```

Comment:
Why has this been changed to a method call? Separating the routing rules out makes it harder to read.

Reply:
Think I initially changed it cause it thought it would have been more than just an `if` but you're right, I have changed these to be inline.

Reply:
The same thing applies to a lot of other places where question variants have been split out into a method rather than kept inline.

Reply:
Has been changed now.